### DISCUSSION TITLE
Is it possible to create 'Cases' from Sales Hub?

### DISCUSSION THREAD

**Discussion by igorlfs** [AleksandrRogov/DynamicsWebApi#167] [Q&A] [answered]
Hello! Thanks for the library! Is it possible to create 'Cases' from Sales Hub? I've tried using the `create` function with collection set to 'cases', but I got an error: \`Resource not found for the segment 'cases'.\`, so It doesn't seem to be the right approach. Am I missing something?

**Comment by AleksandrRogov** [comment, marked as answer]
Hi @igorlfs , I think the entity set name for Case table is `incidents`. Try that and let me know.

```ts
const case = {
  title: "test"
};

const result = await dynamicsWebApi.create({
  collection: 'incidents',
  data: case
});
```

Thank you!
